.azure-devops/create-release.yml

```diff
@@ -21,14 +21,13 @@ parameters:
   - '3.6.x'
   - '3.8.x'
   - '3.9.x'
-- name: pythonVersionTestRun
-  type: string
-  default: '3.8.x'
-  values:
-  - '3.6.x'
-  - '3.8.x'
-  - '3.9.x'
-  - '3.10.x'
+- name: pythonVersionsForTesting
+  type: object
```

Comment:
What are the advantages of using `type: object` here?

Reply:
The type 'object' allows us to pass a list of python versions for which we want to run tests